### DISCUSSION TITLE
New Post(s) Notification on Specific Subreddit

### DISCUSSION THREAD

**Discussion by AndhikaWB** [Docile-Alligator/Infinity-For-Reddit#920] [Ideas]
Just as the title said, it's really helpful to me since certain subreddit (e.g. r/FreeGameFindings) usually do giveaways that require me to claim it asap because it can expire/out of stock at any time.

**Comment by Docile-Alligator** [comment]
Uh this is difficult to implement and I will add other features first.

**Comment by AndhikaWB** [comment]
It's okay. Take your time.

**Comment by benthetechguy** [comment]
I second this, for the same reason.
